src/main/java/com/gluonhq/substrate/target/LinuxTargetConfiguration.java

```diff
@@ -122,6 +123,22 @@ public boolean link() throws IOException, InterruptedException {
         return linkFlags;
     }
 
+    @Override
+    List<String> getTargetSpecificAOTCompileFlags() throws IOException {
+        if (!crossCompile) {
```

Comment:
this is not visible, is it? (It is marked `private`)

Reply:
right, I did that after a codacy compaint but probably a wrong suggestions.